<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>679</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thenga: Advanced ADB GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources1.qrc">
    <normaloff>:/icons/res/debug.png</normaloff>:/icons/res/debug.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Shell</string>
    </property>
    <addaction name="actionAdb_Shell"/>
    <addaction name="actionManage"/>
    <addaction name="actionReboot_Device"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionCascade_Windows"/>
    <addaction name="actionTile_Windows"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdb_Shell"/>
   <addaction name="separator"/>
   <addaction name="actionFile_Browser"/>
   <addaction name="actionManage"/>
   <addaction name="actionSettings"/>
   <addaction name="actionExit"/>
   <addaction name="actionCPU_Stats"/>
  </widget>
  <action name="actionManage">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/settings.png</normaloff>:/icons/res/settings.png</iconset>
   </property>
   <property name="text">
    <string>Manage ADB</string>
   </property>
   <property name="toolTip">
    <string>Start, Stop ADB Server</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/exit.png</normaloff>:/icons/res/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/info.png</normaloff>:/icons/res/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/develop.png</normaloff>:/icons/res/develop.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Change the settings</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAdb_Shell">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/new_tab.png</normaloff>:/icons/res/new_tab.png</iconset>
   </property>
   <property name="text">
    <string>Adb Shell</string>
   </property>
   <property name="toolTip">
    <string>Opens the ADB Shell</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionFile_Browser">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/folder.png</normaloff>:/icons/res/folder.png</iconset>
   </property>
   <property name="text">
    <string>File Browser</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/close.png</normaloff>:/icons/res/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close the current Window</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/close.png</normaloff>:/icons/res/close.png</iconset>
   </property>
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="toolTip">
    <string>Close All windows</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/copy.png</normaloff>:/icons/res/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/paste.png</normaloff>:/icons/res/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCPU_Stats">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/connect.png</normaloff>:/icons/res/connect.png</iconset>
   </property>
   <property name="text">
    <string>CPU Stats</string>
   </property>
  </action>
  <action name="actionCascade_Windows">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/copy.png</normaloff>:/icons/res/copy.png</iconset>
   </property>
   <property name="text">
    <string>Cascade Windows</string>
   </property>
  </action>
  <action name="actionTile_Windows">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/cut.png</normaloff>:/icons/res/cut.png</iconset>
   </property>
   <property name="text">
    <string>Tile Windows</string>
   </property>
  </action>
  <action name="actionReboot_Device">
   <property name="icon">
    <iconset resource="../../Resources1.qrc">
     <normaloff>:/icons/res/usb_green.png</normaloff>:/icons/res/usb_green.png</iconset>
   </property>
   <property name="text">
    <string>Reboot Device</string>
   </property>
   <property name="toolTip">
    <string>Reboot Device</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Resources1.qrc"/>
 </resources>
 <connections/>
</ui>
